grid-proxy/tests/queries/node_test.go

```diff
@@ -130,6 +130,16 @@ var nodeFilterRandomValueGenerator = map[string]func(agg NodesAggregate) interfa
 			return nil
 		}
 
+		runesList := []rune(region)
```

Comment:
seems like that's also duplicated from the code above?

Reply:
yes, it is duplicated same as CityContains and CountryContains filters for both nodes/farms.

basically this snippet tries to get a substring of a random city/country/region from the aggregated values from the database, to pass it the filter and make sure it can search with any part of the value